### PR TITLE
increasing resolution for ticks

### PR DESCRIPTION
**Pull request by Engid**
I hope I did the PR correctly! Its my first time doing it on github.. 

I fixed the tests you mentioned and just changed the values to match what was being output by the functions.

Let me know if I missed anything!

### CODE REVIEW

File: core/src/time/drift_correction/ticks.rs [hunk 1]
```diff
@@ -87,14 +87,14 @@ mod test {
...
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(627));
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(628));
+    assert_eq!(ticks, TicksTime::new(4608000));
     let ticks = correction.next(100);
-    assert_eq!(ticks, TicksTime::new(627));
+    assert_eq!(ticks, TicksTime::new(4608000));
```
Comment (chris-zen) [inline core/src/time/drift_correction/ticks.rs]: Hmmm, I see we'll need to pick some other example of Signature/Tempo/SampleRate to force some error correction (you might want to use the ones from the previous test). Given the increased resolution the correction doesn't trigger in the test anymore. The trick of this test is to show that an error gets accumulated and corrected by seeing a different value every n steps (i.e. 628, 628, 627, 628, ...).

File: core/src/time/ticks.rs [hunk 2]
```diff
@@ -5,7 +5,7 @@ use std::{
 
 use crate::time::{clock, ClockTime, Signature, Tempo};
 
-pub const TICKS_RESOLUTION: u64 = 2*2*3*3*5*5*7*11; //34650
+pub const TICKS_RESOLUTION: u64 = 508032000; // 2^10 * 3^4 * 5^3 * 7^2
```
Comment (chris-zen) [inline core/src/time/ticks.rs:8]: 👍

File: core/src/time/drift_correction/ticks.rs (hunk 1 above)
Comment (chris-zen) [reply core/src/time/drift_correction/ticks.rs]: Look, this is the test adapted to the new resolution. Felt lazy to create a PR on your PR 😉 Could you please modify it in your branch ? Then everything will be ready for merging.
```rust
  #[test]
  pub fn ticks_drift_correction_next() {
    let mut correction = TicksDriftCorrection::new(Signature::new(6, 11), Tempo::new(131), 44100);
    for _ in 0..4 {
      let ticks = correction.next(1000);
      assert_eq!(ticks, TicksTime::new(36584727));  
    }
    let ticks = correction.next(1000);
    assert_eq!(ticks, TicksTime::new(36584728));
  }
```

Comment (Engid) [reply core/src/time/drift_correction/ticks.rs]: Thanks Chris, I pushed that change. Github is saying there are some conflicts in
- core/src/time/drift_correction/ticks.rs
- core/src/time/ticks.rs

The information it shows is a little confusing for me so I'm not sure how to fix that bit.. It looks like the conflict is between your audio-lockfree branch and my issue branch. Not sure why but maybe I did the PR wrong and didn't request to merge with your master branch? 🤷‍♂

Comment (chris-zen) [reply core/src/time/drift_correction/ticks.rs]: @Engid fixed. I changed to master branch.

**Comment by chris-zen** [comment]
@Engid thank you very much for your first contribution. Looking forward for having you full speed after your semester. I have many ideas in my head to experiment with ;-)

**Comment by chris-zen** [comment]
@Engid I didn't realised that your PR was forked from audio-lockfree branch, and I screwed it up by merging it to master. I reverted the PR.

Could you please prepare another PR ? There are two options:

1) You just create another PR from your current work forked from audio-lockfree, then I will merge into my ongoing work at that branch.
2) You cherry-pick your changes to another fork from master, then I will merge it into master.

Sorry for the inconveniences.

**Pull request merged**